src/pb-login.js

```diff
@@ -268,9 +269,31 @@ export class PbLogin extends pbMixin(LitElement) {
         this.emitTo('pb-login', resp);
     }
 
+    /**
+     * 
+     * @param {Array<String>} arr array containg string values (name of groups)
+     * @param {String} val value to check if it's in the array
+     * @returns true if the checked values is in the array
+     */
+    _isItemInArray(arr, val) {
+        return arr.some((arrVal) => val === arrVal);
+    }
+    
+    /**
+     * 
+     * @param {object} info object returned by login function; 
+     * contains groups the user is a member of
+     * @returns true if user is member of one of defined groups
+     */
     _checkGroup(info) {
         if (this.group) {
-            return info.groups && info.groups.indexOf(this.group) > -1;
+            let groupArray = this.group.split(/\W+/);
```

Comment:
Using any punctuation would not allow valid group names like `tei-demo` or similar. Maybe limit it to whitespace or comma?